book-1-orientation/chapters/STATIC_CLASSES.md

```diff
@@ -0,0 +1,106 @@
+# Static classes, methods and properties
+
+In C# a `class` normally acts as a blueprint for creating objects within your program. However, there are times when you don't need to create a new object in order get your work done. Sometimes you just need a function or a property without the "context" of an object.
+
+This is where the `static` classes come in.
+
+Consider the following:
+
+```csharp
+public static class MathUtils {
+    public static double PI {
+        get {
+            return 3.14159;
+        }
+    }
+
+    public static int Pow(int num, int exponent) {
+        int result = 1;
+        for(int i=0; i<exponent; i++) {
+            result = result * num;
+        }
+        return result;
+    }
+}
+```
+
+Note the use of the `static` keyword in the code above.
+
+When applied to a `class` (as in `public static class MathUtils`), it tells C# that the class **cannot** be instantiated. Meaning it is impossible to make a `new MathUtils()` object.
+
+When applied to a `property` or `method` the `static` keyword tells C# that you can use the property or method without an instance of a class. That is, without an object.
+
+So, if we don't have an object, how do we use static properties and methods?
+
+```cs
+static void Main(string[] args) {
+    int eight = MathUtils.Pow(2, 3);
+    double pi = MathUtils.PI;
+
+    Console.WriteLine($"I {eight} some {pi}.");
```

Comment:
You could even mention that `Console.WriteLine()` itself is a static method since we do not need to use `new Console()` before using it.

Reply:
I added a short section to include this.